README.md

```diff
@@ -3956,7 +3956,7 @@ It's in a **.zip** file, so it will need a little extra preprocessing:
 # fetch from Web, and store the downloaded data to the file 'bike.sip'
 > download.file('https://archive.ics.uci.edu/ml/machine-learning-databases/00275/Bike-Sharing-Dataset.zip','bike.zip')
 > unzip('bike.zip')  # out come the files 'day.csv' and 'hour.csv'
-> day <- read.csv('day.csv',header=TRUE)
+> day <- read.csv('day.csv',header=TRUE, stringsAsFactors=TRUE)
```

Comment:
Newer versions of R doesn't do this automatically, while the rest of the code assumes some of the variables are factors.